components/select_team/select_team.tsx

```diff
@@ -190,7 +201,7 @@ export default class SelectTeam extends React.PureComponent {
                 </div>
             );
         } else {
-            let joinableTeamContents = [];
+            let joinableTeamContents: any = [];
```

Comment:
I think a `ReactNode` would be more appropriate here, but I guess `any` might be okay for now as the array/forEach logic would have to be replaced?

Reply:
`joinableTeamContents` is used both as an array

```javascript
joinableTeamContents.push(
   key={'team_' + listableTeam.name}
   team={listableTeam}
   ...                        
);                    
```

and to store JSX

```javascript
  joinableTeamContents = (
     <div className='signup-team-dir-err'>
         ...
     </div>
  );                           
```

Reply:
I think `ReactNode` can also take an array as well as JSX, but I guess that doesn't work when we call `.length` on it below